client/state/ui/guided-tours/contexts.js

```diff
@@ -26,6 +27,9 @@ export const isNewUser = state => {
 	return ( Date.now() - creation ) <= WEEK_IN_MILLISECONDS;
 };
 
+export const userHasInteractedWithComponent = componentName => state =>
+	getLastAction( state ).component === componentName;
+
```

Comment:
This is obviously subjective and debatable, but I think it's easier to achieve consistency if our naming style is more like `hasUserInteractedWithComponent`. The same idea would apply to the following selector in this file (`isSelectedSitePreviewable`).

It makes the return type immediately obvious (the func is a predicate), and would mean a more coherent global list of selectors in `state`. (/cc @aduth)